tstest/integration/vms/vms_test.go

```diff
@@ -463,6 +525,77 @@ func TestVMIntegrationEndToEnd(t *testing.T) {
 	}
 }
 
+func copyBinaries(t *testing.T, conn *ssh.Client) {
+	bins := integration.BuildTestBinaries(t)
+
+	cli, err := sftp.NewClient(conn)
+	if err != nil {
+		t.Fatalf("can't connect over sftp to copy binaries: %v", err)
+	}
+
+	mkdir(t, cli, "/usr/bin")
+	mkdir(t, cli, "/usr/sbin")
+	mkdir(t, cli, "/etc/systemd/system")
+	mkdir(t, cli, "/etc/default")
+
+	copyFile(t, cli, bins.Daemon, "/usr/sbin/tailscaled")
+	copyFile(t, cli, bins.Cli, "/usr/bin/tailscale")
+	copyFile(t, cli, "../../../cmd/tailscaled/tailscaled.defaults", "/etc/default/tailscaled")
```

Comment:
`../../../` seems prone to breakage in case of any directory refactoring. Maybe something like [this](https://stackoverflow.com/questions/31873396/is-it-possible-to-get-the-current-root-of-package-structure-as-a-string-in-golan) would work. Adding TODO is fine.